### PR TITLE
Auto-create config.ini if it doesn't exist

### PR DESCRIPTION
**Pull request by NotActuallyTerry**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -20,15 +20,17 @@
...
+        'client_id': '',
+        'client_secret': '',
+        'bot_token': '',
+        'bot_owner': '',
+        'bot_server': ''
+        }
+    config.write(open(config_path, 'w'))
     exit(1)
-
+
```
Comment (thomotron) [inline bot.py]: Can you remove the whitespace?

Comment (NotActuallyTerry) [reply bot.py]: no

Comment (NotActuallyTerry) [reply bot.py]: ok fine

**Comment by thomotron** [comment]
cheers

**Pull request merged**